src/lib/analytics.ts

```
@@ -0,0 +1,310 @@
+    
+    // Also track as a custom event for easier filtering
+    posthog.capture("game_event", {
+      event_type: eventName,
+      ...enhancedProperties,
+    });
+  };
+
+  /**
+   * Identify the player with session information
+   */
+  const identifyPlayer = (sessionId: string, properties?: Record<string, any>): void => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)